themes/theme-gmd/components/ProductGrid/components/Item/components/ItemDetails/index.jsx

```diff
@@ -37,7 +37,15 @@ class ItemDetails extends PureComponent {
     return (
       <div className={styles.details}>
         <ItemName display={display} productId={productId} name={name} />
+
+        {/* This feature is currently in BETA testing.
+        It should only be used for approved BETA Client Projects */}
         <MapPriceHint productId={productId} />
+
+        {/* This feature is currently in BETA testing.
+        It should only be used for approved BETA Client Projects */}
+        <OrderQuantityHint productId={productId} />
```

Comment:
We expose the `ProductGrid` through the `ThemeContext`. So there will be the same console errors like at the `ProductCard` when a shop is deployed with the `theme-config` extension and one which provides a custom route that uses the `ProductGrid`.

Same for the ios theme.